crates/red_knot_python_semantic/src/semantic_index/re_exports.rs

```diff
@@ -279,11 +310,11 @@ impl<'db> Visitor<'db> for ExportFinder<'db> {
     }
 }
 
-struct WalrusFinder<'a, 'db> {
-    export_finder: &'a mut ExportFinder<'db>,
+struct WalrusFinder<'a, 'b, 'db> {
```

Comment:
Do we need the second lifetime here or could we use `'a` in places of both `'a` and `'b` (without the borrow checker getting upset?